data_structures/binary_tree/binary_tree_mirror.py

```diff
@@ -0,0 +1,44 @@
+# Created by susmith98
+
+
+"""
+Problem Description:
+Given a binary tree, return it's mirror.
+"""
+
+
+def binaryTreeMirrorDict(
```

Comment:
```suggestion
def binary_tree_mirror_dict(
```
See the discussion on Python naming in CONTRIBUTING.md.